Products/CMFPlone/browser/login/login.py

```diff
@@ -104,7 +104,7 @@ def updateWidgets(self):
 
         if self.use_email_as_login():
             self.widgets[fieldname_name].label = _(u'label_email',
-                                                   default=u'Email')
+                                                   default=u'E-mail')
```

Comment:
Proof me wrong, but usually in English it is written `Email`` even if e-mail is allowed too:
https://en.oxforddictionaries.com/definition/email

Reply:
I was also searching for the correct way and it seems that both are OK... The point of the PR was that we are already using "E-mail" somewhere else for long time (https://github.com/plone/plone.app.portlets/blob/master/plone/app/portlets/portlets/login.pt#L48 or https://github.com/plone/plone.app.users/blob/master/plone/app/users/schema.py#L67 for example), and the point was to unify them all changing this one, because it's the latest thing that was added here.

Reply:
Lets ask some native speakers? Ping @tkimnguyen @MatthewWilkes

Reply:
As @jensens says, both are acceptable but "email" is more common these days. That said, I agree with @erral's logic, it's better to use the slightly less common (but still correct) form everywhere than to have a mixture.